client/src/components/MovieCard/CardMenuData/CardMenuData.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC } from 'react';
+import { styled } from '@mui/material/styles';
+import { Box, MenuItem } from '@mui/material';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import FormatListBulletedIcon from '@mui/icons-material/FormatListBulleted';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import { Movie } from '../../typedefs/typedefs';
+import { CardMenu } from '../CardMenu/CardMenu';
+
+const MenuItems = styled(MenuItem)(() => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+}));
+
+interface Props {
+  movie: Movie,
+  onDeleteCard: () => void,
+  isAdded: (movie: Movie) => boolean;
+  onCardSelect: (movie: Movie) => void,
+  onSelectMovie: (movie: Movie) => void;
```

Comment:
move to the upper component
```suggestion
  isMovieAdded: () => boolean;
  onCardSelect: () => void,
  onSelectMovie: () => void;
```